src/cephadm/cephadm

```diff
@@ -129,6 +129,10 @@ class Monitoring(object):
             "memory": "1GB",
             "args": [
                 "--no-collector.timex",
+                "--path.rootfs=/rootfs",
+                "--path.sysfs=/host/sys",
+                "--path.procfs=/host/proc",
+                "'--collector.filesystem.ignored-mount-points=^/(sys|proc|dev|host|etc)($$|/)'"
```

Comment:
We do something similar to this when deploying node-exporter container with ceph-ansible. I don't see any bind mounts for the container defined here though. Also, I don't know that much about what this does but it seems confusing that `rootfs` would be `/rootfs` while other options like `sysfs` and `procfs` point to `/host/<something>`.

Reply:
Yes, seems strange to me too: Shouldn't `rootfs` be `/host` for this to make sense?

Reply:
That's why i created https://github.com/ceph/ceph/pull/34347.

Reply:
> Yes, seems strange to me too: Shouldn't `rootfs` be `/host` for this to make sense?

Not at the moment because `/` is mounted to `/rootfs` at the moment. But with the mentioned PR above this could be simplified.

Reply:
https://github.com/ceph/ceph/pull/34347#issuecomment-616863343

Reply:
> I don't see any bind mounts for the container defined here though.

The bind mounts already exist in the source: https://github.com/ceph/ceph/blob/5100974c0abd1c45b5e139a3e5276decd9707e28/src/cephadm/cephadm#L1444-L1446

> Also, I don't know that much about what this does but it seems confusing that rootfs would be /rootfs while other options like sysfs and procfs point to /host/<something>.

> Yes, seems strange to me too: Shouldn't rootfs be /host for this to make sense?

> That's why i created #34347 some time ago.

Unfortunately, I'm also not able to explain why this is done this way. But there seems to be an unacceptable [difference in behavior](https://github.com/ceph/ceph/pull/34347#pullrequestreview-402777771) when just "rootfs" is used to bind mount and passed as argument to node-exporter, which I found when testing the suggested simplification. This PR however seems to function as it should.

Reply:
Well, minus the selinux errors..